### ISSUE TITLE
[BETA v3.4.0] org.freedesktop.DBus.Error.InteractiveAuthorizationRequired error in system log

### ISSUE THREAD

**Issue by jpkelly** [bitfocus/companion-pi#31]
In the latest beta (v3.4.0+7240-main-c87f758d) I am seeing the following in system log when Companion is running.

`Aug 31 19:43:11 pi dbus-daemon[632]: [system] Rejected send message, 0 matched rules; type="error", sender=":1.15074" (uid=0 pid=331813 comm="/lib/systemd/systemd-hostnamed") interface="(unset)" member="(unset)" error name="org.freedesktop.DBus.Error.InteractiveAuthorizationRequired" requested_reply="0" destination=":1.16604" (uid=1001 pid=365375 comm="hostnamectl --json short")`

This is on a Pi 5 running Debian Bookworm

**Comment by Julusian** [comment]
Yeah I can see this too, but I can't explain why that happens and it doesn't appear to break anything.

I'm not going to keep looking into this, as google isn't finding anything helpful and it appears pretty harmless, other than the log noise.
If someone has any ideas on how to resolve it, that would be appreciated
